webapp/src/components/viewHeader/newCardButtonTemplateItem.tsx

```diff
@@ -12,30 +12,44 @@ import EditIcon from '../../widgets/icons/edit'
 import OptionsIcon from '../../widgets/icons/options'
 import Menu from '../../widgets/menu'
 import MenuWrapper from '../../widgets/menuWrapper'
+import CheckIcon from '../../widgets/icons/check'
+import {BoardView} from '../../blocks/boardView'
 
 type Props = {
+    boardView: BoardView
     cardTemplate: Card
     addCardFromTemplate: (cardTemplateId: string) => void
     editCardTemplate: (cardTemplateId: string) => void
 }
 
 const NewCardButtonTemplateItem = React.memo((props: Props) => {
-    const {cardTemplate} = props
+    const {boardView, cardTemplate} = props
     const intl = useIntl()
     const displayName = cardTemplate.title || intl.formatMessage({id: 'ViewHeader.untitled', defaultMessage: 'Untitled'})
+
+    const icon = boardView.defaultTemplateId === cardTemplate.id ? <CheckIcon/> : cardTemplate.icon
+
     return (
         <Menu.Text
             key={cardTemplate.id}
             id={cardTemplate.id}
             name={displayName}
-            icon={<div className='Icon'>{cardTemplate.icon}</div>}
```

Comment:
The issue with this approach is it replaces the template's icon. Suggest the following instead:
* Add a new CSS style to make a menu item bold
* Pass that as the className on the Menu.Text option if it is the default template

Reply:
I got you. Let me check your method

Reply:
@chenilim Should I also add a color to tag it? I test the bold font, but not very obvious to see it